### PR TITLE
WIP fix(helm): add svc definition for 'helm init'

### LINKED ISSUE #648: tiller pod not fronted by a kubernetes service
After running a `helm init` I see:

```
$ kubectl get svc,rc,po --namespace=helm
NAME              DESIRED   CURRENT   AGE
tiller-rc         1         1         29m
NAME              READY     STATUS    RESTARTS   AGE
tiller-rc-32koc   1/1       Running   0          29m
```

But the rc is not fronted by a service so if this is a remote kubernetes cluster, there's no way for the `helm` cli to talk to the tiller server. 😢

### PR DESCRIPTION
**Pull request by technosophos**
Closes #648.

### CODE REVIEW

File: pkg/client/install.go [hunk 1]
```diff
@@ -85,4 +85,20 @@ spec:
...
+  name: tiller-svc
+  namespace: helm
+spec:
+  selector:
+    app: helm
+    name: tiller
+  ports:
+    - protocol: TCP
+      port: 43144
+      targetPort: 43144
```
Comment (sgoings) [inline pkg/client/install.go]: Why is this not 8080?

Comment (technosophos) [inline pkg/client/install.go]: gRPC binary protocol, not REST/JSON

**Comment by adamreese** [comment]
What would the proxy url be to the service?
`http://192.168.64.94:8080/api/v1/proxy/namespaces/helm/services/tiller-svc`

**Comment by technosophos** [comment]
Currently I am waiting on this while @adamreese experiments on some other ideas.

**Comment by technosophos** [comment]
The new tunnel logic makes this largely irrelevant

**Pull request closed without merging**